ui/tools/objectinspector/objectinspectorwidget.h

```diff
@@ -41,6 +41,7 @@
 #include "methodsextensionclient.h"
 #include "connectionsextensionclient.h"
 #include "applicationattributetab.h"
+#include "../bindinginspector/bindingtab.h"
```

Comment:
I would prefer you add bindinginspector in the objectinspector include paths instead of adding there and there relative file paths across plugins.